network/p2p/p2p_sync.go

```diff
@@ -75,8 +73,8 @@ func (n *p2pNetwork) LastDecided(logger *zap.Logger, mid spectypes.MessageID) ([
 	if !n.isReady() {
 		return nil, p2pprotocol.ErrNetworkIsNotReady
 	}
-	pid, peerCount := n.fork.ProtocolID(p2pprotocol.LastDecidedProtocol)
-	peers, err := n.getSubsetOfPeers(logger, mid.GetPubKey(), peerCount, allPeersFilter)
+	pid, maxPeers := n.fork.ProtocolID(p2pprotocol.LastDecidedProtocol)
+	peers, err := waitSubsetOfPeers(logger, n.getSubsetOfPeers, mid.GetPubKey(), 3, maxPeers, time.Second*24, allPeersFilter)
```

Comment:
IMO `3` and `time.Second*24` should be consts or at least vars, solely for the purpose of them being named.